src/overlay/Peer.h

```diff
@@ -112,6 +116,18 @@ class Peer : public std::enable_shared_from_this<Peer>,
         xdr::msg_ptr mMessage;
     };
 
+    struct QueuedOutboundMessage
+    {
+        StellarMessage mMessage;
```

Comment:
we should find a way to not copy messages - I suspect this adds quite a bit of overhead (or we need to move things properly?)